### PR TITLE
Added second header to index

### PR DESCRIPTION
**Pull request by SSEDOR**
Pls review and update

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <head><title>My Test Page</title></head>
 <body>
 <h1>A Simple Web Example</h1>
+<h2>Testing Out New Features</h2>
```
Comment (madtypist) [inline index.html]: Please use h3 tag, not h2

**Review by madtypist — CHANGES_REQUESTED** [review]
For style purposes I arbitrarily have decided I want you to use h3 tags instead of h2

File: index.html (hunk 1 above)
Comment (SSEDOR) [reply index.html]: Changes have been made

**Review by madtypist — APPROVED** [review]
Looks great! Thanks for the change.

**Pull request merged**